config/dockergento/grumphp/hooks/pre-commit

```diff
@@ -20,8 +21,14 @@ DOCKER_PHP_CONTAINER_ID=$(docker-compose -f ${DOCKER_COMPOSE_FILE} ps -q phpfpm)
 # Remove single quotes from hook_command. It is needed to use it inside the docker exec
 HOOK_COMMAND=$(echo "$(HOOK_COMMAND)" | sed "s/'//g")
 
+echo ${DIFF} >> grumphp-diff && docker cp  grumphp-diff ${DOCKER_PHP_CONTAINER_ID}:/tmp/grumphp-diff
```

Comment:
White space same as before

Reply:
If you want you can add a variable for `grumphp-diff` filename as you had before